server/server_test.go

```diff
@@ -1170,6 +1170,33 @@ func TestIndexGetEscaped(t *testing.T) {
 
 }
 
+func TestVersionGet(t *testing.T) {
+
+	/*
+		Example of GET version:
+
+		Version: 0.9.3-dev
+		Build Commit: c5171ed5-dirty
+		Build Timestamp: 2018-10-07T02:12:58Z
+		Build Hostname: JOHNDOE-M-9130
+	*/
+
+	f := newFixture(t)
+
+	get := newReqV1(http.MethodGet, "/data/system/version", "")
+	f.server.Handler.ServeHTTP(f.recorder, get)
+	if f.recorder.Code != 200 {
+		t.Fatalf("Expected 200 OK but got %v", f.recorder)
+		return
+	}
+
+	page := f.recorder.Body.String()
+	if !strings.Contains(page, "Version: ") {
```

Comment:
This test will be modified a bit with the JSON change above. It would be good to check that the API response matches the `version.Version`, `version.BuildCommit`, etc.